get-started/hallucination-agent/question-answer.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Question - answer 
+description: Learn how to use the kluster.ai Hallucination Detection API to validate the truthfulness of answers to questions.
+---
+
+# Question/Answer
+
+The Question/Answer method allows you to validate whether an answer to a specific question contains hallucinated information. This approach is ideal for fact-checking individual responses against provided context or general knowledge.
+
+## Prerequisites
+
+Before getting started with fine-tuning, ensure you have the following:
+
+- **kluster.ai account** - sign up on the [kluster.ai platform](https://platform.kluster.ai/signup){target=_blank} if you do not have one
+- **kluster.ai API key** - after signing in, go to the [API Keys](https://platform.kluster.ai/apikeys){target=_blank} section and create a new key. For detailed instructions, see the [Get an API key](https://docs.kluster.ai/get-started/get-api-key/){target=_blank} guide
+
+
+## How it works
+
+The service evaluates the truthfulness of an answer to a question by:
+
+1. Analyzing the original question or prompt.
+2. Examining the provided answer.
+3. Determining if the answer contains hallucinated or unsupported information.
+4. Providing a detailed explanation of the reasoning behind the determination as well as the search results used for fact-checking.
+
+### Parameters
+
+| Parameter | Type | Required | Description |
+| :---: | :---: | :---: | :---: |
+| `prompt` | string | Yes | The question asked or instruction given. |
+| `output` | string | Yes | The answer to verify for hallucinations. |
+| `context` | string | No | Optional reference material to validate against. |
+| `return_search_results` | boolean | No | Whether to include search results (default: false). |
+
+
+The API returns a JSON object with the following structure:
+
+```json
+{
+    "is_hallucination": boolean,
+    "usage": {
+        "completion_tokens": number,
+        "prompt_tokens": number,
+        "total_tokens": number
+    },
+    "explanation": "string",
+    "search_results": []  // Only included if return_search_results is true
+}
+```
+## Usage modes
+
+The agent operates in two distinct modes depending on whether you provide context with your request:
+
+- **Fact-checking mode** - when no context is provided, the agent verifies answers against general knowledge and external sources.
```

Comment:
```suggestion
- **Fact-checking mode**: When no context is provided, the agent verifies answers against general knowledge and external sources.
```

Reply:
Maybe briefly mention how is context provided or not provided? so that is dead obvious